src/api/utils.ts

```diff
@@ -18,12 +18,14 @@ export const ErrorResponse = z.object({
 });
 export type ErrorResponse = z.infer<typeof ErrorResponse>;
 
-export type ErrorType =
-  | 'unknown'
-  | 'default'
-  | 'network-error'
-  | 'timeout'
-  | 'cancel';
+type ErrorType = 'unknown' | 'default' | 'network-error' | 'timeout' | 'cancel';
+
+export type AxiosErrorKind =
```

Comment:
There were quite a few places that accessed the axios error type directly. Since we now intercept the error and make it an `ErrorResponse` it won't be possible any more. We do set the `kind` to a matching error type, so we can do something like this. 🤷